app/assets/stylesheets/active_admin/components/_tables.scss

```diff
@@ -16,9 +20,8 @@ table.index_table {
   border-spacing: 0;
 
   th {
-    @include section-header;
+    background-color: #e9e9e9;
```

Comment:
Color literals like `#e9e9e9` should only be used in variable declarations; they should be referred to via variable everywhere else.